docs/MODDING.md

```diff
@@ -0,0 +1,191 @@
+# Modding
+
+> So you would like to create a code mod? _BUCKLE UP_
+
+## Preface
+
+Git is required to be installed. Knowing how to use git is going to help, I will list out commands that should set you on the right but without a general understanding you will likely get stuck if you deviate from the happy path.
+
+General coding knowledge is also usually a requirement. You might be able to get by without, but the more knowledge-able the better, as it will allow you to find what you are looking for and troubleshoot much more easily.
+
+## What the fork?
+
+Your first step is to fork the repository. To do this, you will need a github account. Assuming you have a github account you can navigate to the Shipwright repo [here](https://github.com/HarbourMasters/Shipwright) and press the `Fork` button in the top right of the screen. When that process is complete you should land on a page that looks similar to the original repo, but under your username (eg: `https://github.com/<GITHUB USERNAME>/Shipwright`).
+
+On this page you should see a green `Code` button, click this and copy the URL within. (You may use the github desktop app here as well, but I will not provide instructions for it). Then in your terminal/command prompt you will git clone to a local development folder using the copied URL
+
+```bash
+cd ~/Documents
+git clone https://github.com/<GITHUB USERNAME>/Shipwright.git
+```
+
+At this point, I will now direct you to our [[BUILDING]] guide, it will have the most up to date documentation on getting the Ship running on your local machine. Once you have successfully built and launched the executable you may return here.
```

Comment:
Not sure how relative links work with markdown but this should do.
```suggestion
At this point, I will now direct you to our [BUILDING](building.md) guide, it will have the most up to date documentation on getting the Ship running on your local machine. Once you have successfully built and launched the executable you may return here.
```